samples/CommunityToolkit.Maui.Sample/Views/Popups/XamlBindingPopup.xaml.cs

```diff
@@ -1,8 +1,14 @@
+using CommunityToolkit.Maui.Sample.Models;
 using CommunityToolkit.Maui.Views;
 
 namespace CommunityToolkit.Maui.Sample;
 
 public partial class XamlBindingPopup : Popup
 {
-	public XamlBindingPopup() => InitializeComponent();
+	public XamlBindingPopup(PopupSizeConstants popupSizeConstants)
```

Comment:
Can we have one example without PopupSize?
What will happen if we don't set the size? I expect to see Medium size popup